docs/integrations.md

```diff
@@ -34,53 +34,58 @@ Zafira uses integration with Amazon S3 service to store user profile photos and
 * Grant read/write permissions for new user (read [detailed guide](https://aws.amazon.com/blogs/security/writing-iam-policies-how-to-grant-access-to-an-amazon-s3-bucket/))
 * Generate new access/secret keys
 * Turn on Amazon integration and provide access/secret keys and name of the bucket you've created
```

Comment:
Enable Amazon integration and provide access / secret keys and the name of the bucket you created.